tools/validators/instance_validator/validate/handler.py

```diff
@@ -159,13 +159,6 @@ def RunValidation(filenames: List[str],
             'run telemetry validation. See here for more details: '
             'https://google.github.io/digitalbuildings/tools/validators/'
             'instance_validator/#telemetry-validation')
-  # pylint: disable=broad-except
-  # Using broad exception to catch any underlying error that might cause
-  # validation to fail
-  except Exception as ex:
```

Comment:
Why remove this broad catch block? without it, wouldn't the output be something like:

"<exception text>
[INFO] Instance validation completed."

which is confusing

Reply:
it will output "instance validation complete", but it will also print the error stack. This info just shows it completed.

there is a separate message that tells the user whether validation succeeded: "[INFO]\tAll entities validated SUCCESSFULLY."

Reply:
Best solution would be to open a bug to write more detailed exception handling for the handler to catch a wide variety of errors. For now, I'm okay with this change.